canvg.js

```diff
@@ -231,8 +231,14 @@
 			}
 			else if (window.DOMParser)
 			{
-				var parser = new DOMParser();
-				return parser.parseFromString(xml, 'text/xml');
+				try {
+					var parser = new DOMParser();

```

Comment:
This was my experience while cross browser testing. I Believe it was one of the older version of MSIE I tested where a call to parseFromString after such a call had thrown an error would throw another error. Using a second instance of DOMParser prevents this. 
Thanks for the quick reply.